maintainers/maintainer-list.nix

```diff
@@ -23638,5 +23638,13 @@
     githubId = 59917878;
     name = "Mathias Zhang";
   };
+  BonusPlay = {
+    name = "Bonus";
+    email = "nixos@bonusplay.pl";
+    matrix = "@bonus:bonusplay.pl";
+    github = "BonusPlay";
+    githubId = 8405359;
+    keys = [ { fingerprint = "8279 6487 A4CA 2A28 E8B3  3CD6 C7F9 9743 6A20 4683"; } ];
+  };
 }
 # Keep the list alphabetically sorted.
```

Comment:
^ you should probably move this a bit further up

Reply:
yep, working on it :+1: